README.md

```diff
@@ -6,15 +6,32 @@ SourceTracker was originally described in [Knights et al., 2011](http://www.ncbi
 If you use this package, please cite the original SourceTracker paper linked
 above pending publication of SourceTracker 2.
 
+# File Formats
+
+For descriptions of all file formats and options, please see the help
+documentation, available with the command `sourcetracker2 gibbs --help`. 
+
+In brief, this script requires a feature X sample contingency table
+(traditionally an OTU table) and sample metadata (traditionally a mapping file).
+Any feature table that can be read by the `biom` package will be acceptable
```

Comment:
Note that backward compatibility won't be guaranteed >= 2.2.0, so be sure to include that as an upper boundary.